main.rb

```diff
@@ -1,4 +1,60 @@
 require_relative 'app'
 
-app = App.new
-app.run
+class Main
+  def initialize()
+    @app = App.new
+    @hash = { 1 =>
+    proc do
+      @app.list_books
+    end, 2 =>
+    proc do
+      @app.list_people
+    end, 3 =>
+    proc do
+      @app.create_person
+    end, 4 =>
+    proc do
+      @app.create_book
+    end, 5 =>
+    proc do
+      @app.create_rental
+    end, 6 =>
+    proc do
+      @app.list_rentals
+    end, 7 => end_hash }
+  end
+
+  def display_menu
+    puts 'Welcome to school library App'
+    puts 'Please choose an option to enter in a number'
+    puts '1. List all books'
+    puts '2. List all people'
+    puts '3. Create a person'
+    puts '4. Create a book'
+    puts '5. Create a rental'
+    puts '6. List rentals for a given person id'
```

Comment:
- [ ] Whenever I select option 6 to list the number of rentals for a person the app crashes. Kindly look into that.

  ![image](https://user-images.githubusercontent.com/36542986/226598544-4c6e32cd-9b45-4d86-bc0a-17437a952df0.png)